darts/metrics/metrics.py

```diff
@@ -136,11 +155,10 @@ def _get_values_or_raise(series_a: TimeSeries,
                                                                     series_a.time_index, series_b.time_index),
                  logger)
 
-    # TODO: we may want to change this...
-    series_a_det = series_a_common if series_a_common.is_deterministic else series_a_common.quantile_timeseries(quantile=0.5)
-    series_b_det = series_b_common if series_b_common.is_deterministic else series_b_common.quantile_timeseries(quantile=0.5)
-
-    return series_a_det.univariate_values(), series_b_det.univariate_values()
+    # TODO: we may want to change this... -> (Dennis: 13.09.2021) how about this?
```

Comment:
Looks good :)
You can change the docstring on line 142: `For stochastic series, return some deterministic quantile value`